exp_final_whitebox_attacker.py

```diff
@@ -0,0 +1,53 @@
+import sys
+
+from gbra.data.network_loader import Movielens100kLoader
+from gbra.attackers.attacker import *
+from gbra.recommender.recommenders import PixieRandomWalkRecommender
+
+ITERATIONS = 10
+
+PIXIE_PARAMS = {
+    'n_p': 30,
+    'n_v': 4,
+    'num_steps_in_walk': 200,
```

Comment:
1000?

Reply:
fixed

Reply:
Also with the new merged code this is not `num_steps_in_walk` but `max_steps_in_walk`